### ISSUE TITLE
error when using layouts in pug

### ISSUE THREAD

**Issue by miguelAliaga** [fastify/point-of-view#19]
`App.register(require('point-of-view'), {
  engine: {
    pug: require('pug')
  },
  options:{},
  templates: 'templates'
})`

I got :
`Error: the "filename" option is required to use includes and extends with "relative" paths`
in the file :

`extends ./layout.pug
block content
  p hola`

How I pass filename to options object inside the configuration ?

**Comment by miguelAliaga** [comment]
i have no sintax errors

**Comment by delvedor** [comment]
I don't think I understood very well your issue, can you rephrase?
If you need to pass some custom option to the template engine, you should use the options key.

```js
  fastify.register(require('point-of-view'), {
    engine: {
      pug: require('pug')
    },
    options: {

   
  })
```

**Comment by delvedor** [comment]
Closing this, if you have more questions feel free to reopen :)
